controllers/machinedeletionremediation_controller.go

```diff
@@ -76,25 +91,43 @@ type MachineDeletionRemediationReconciler struct {
 //
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
-func (r *MachineDeletionRemediationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (r *MachineDeletionRemediationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (finalResult ctrl.Result, finalErr error) {
 	log := r.Log.WithValues("machinedeletionremediation", req.NamespacedName)
 
 	log.Info("reconciling...")
 
 	var err error
 	var remediation *v1alpha1.MachineDeletionRemediation
 	if remediation, err = r.getRemediation(ctx, req); remediation == nil || err != nil {
-		if apiErrors.IsNotFound(err) {
```

Comment:
Because `IsNotFound` is already handled inside `getRemediation`

https://github.com/clobrano/machine-deletion-remediation/blob/99704c46dfd12f6219285c2e0796c6caffae4e6c/controllers/machinedeletionremediation_controller.go#L192-L195

Reply:
missed that, sorry 👍🏼